app/controllers/claims/schools/claims_controller.rb

```diff
@@ -1,3 +1,55 @@
 class Claims::Schools::ClaimsController < ApplicationController
   include Claims::BelongsToSchool
+
+  def index
+    render locals: { claims: @school.claim_applications }
+  end
+
+  def new
+    claim = Claim.create!(school_id: params.require(:school_id), draft: true)
```

Comment:
This might be more succinct (thanks to `Claims::BelongsToSchool`):

```suggestion
    claim = @school.create_claim!(draft: true)
```